definitions/scenarios/satellite_upgrade.rb

```diff
@@ -23,14 +23,43 @@ class PreUpgradeCheck < Abstract
       run_strategy :fail_slow
     end
 
+    # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
     def compose
-      add_steps(find_checks(:default))
-      add_steps(find_checks(:pre_upgrade))
+      add_steps(
+        Checks::Foreman::FactsNames.new, # if Foreman database present
+        Checks::ForemanProxy::CheckTftpStorage.new, # if Satellite with foreman-proxy+tftp
+        Checks::ForemanProxy::VerifyDhcpConfigSyntax.new, # if foreman-proxy+dhcp-isc
+        Checks::ForemanTasks::NotPaused.new, # if foreman-tasks present
+        Checks::Puppet::VerifyNoEmptyCacertRequests.new, # if puppetserver
+        Checks::ServerPing.new,
+        Checks::ServicesUp.new,
+        Checks::SystemRegistration.new
+      )
+      add_steps(
+        Checks::CheckHotfixInstalled.new,
+        Checks::CheckTmout.new,
+        Checks::CheckUpstreamRepository.new,
+        Checks::Disk::AvailableSpace.new,
+        Checks::Disk::AvailableSpaceCandlepin.new, # if candlepin
+        Checks::Foreman::ValidateExternalDbVersion.new, # if external database
+        Checks::Foreman::CheckCorruptedRoles.new,
+        Checks::Foreman::CheckDuplicatePermissions.new,
+        Checks::Foreman::TuningRequirements.new, # if katello present
+        Checks::ForemanOpenscap::InvalidReportAssociations.new, # if foreman-openscap
+        Checks::ForemanTasks::Invalid::CheckOld.new, # if foreman-tasks
+        Checks::ForemanTasks::Invalid::CheckPendingState.new, # if foreman-tasks
+        Checks::ForemanTasks::Invalid::CheckPlanningState.new, # if foreman-tasks
+        Checks::ForemanTasks::NotRunning.new, # if foreman-tasks
+        Checks::NonRhPackages.new,
+        Checks::PackageManager::Dnf::ValidateDnfConfig.new,
+        Checks::Repositories::CheckNonRhRepository.new
+      )
       add_step(Checks::CheckIpv6Disable)
       add_step(Checks::Disk::AvailableSpacePostgresql13)
       add_step(Checks::Repositories::Validate.new(:version => target_version))
       add_step(Checks::CheckOrganizationContentAccessMode)
     end
+    # rubocop:enable Metrics/MethodLength, Metrics/AbcSize
```

Comment:
hello :wave:

Reply:
This re-enables those linting rules so it only applies to the above methods